webapp/src/mutator.ts

```diff
@@ -836,6 +836,19 @@ class Mutator {
         )
     }
 
+    async changeViewCardOrder(boardId: string, viewId: string, oldCardOrder: string[], cardOrder: string[], description = 'reorder'): Promise<void> {
```

Comment:
Removing the requirement of the entire view object to change the order or a card.